### ISSUE TITLE
Warrior Skill (Furious Slash) Bugged 

### ISSUE THREAD

**Issue by cassillia** [AshamaneProject/AshamaneCore#894]
**Description:**
Furious Slash
Does not increase Bloodthirst critical strike chance by 15%

**Usefull Wowhead links:**
http://www.wowhead.com/spell=231824/furious-slash

**Comment by HannibalRoG** [comment]
Furious slash already exist in #499.

**Comment by cassillia** [comment]
Yep it works now. I think I dropped the quest because there was nowhere to hand in.

**Comment by cassillia** [comment]
Commanding Shout is still broken. Does not add any health. spell id 97463 should be 97462 I believe

**Comment by cassillia** [comment]
Gore is still not getting triggered. I think the fix made the abilities trigger Gore trigger ( Gory Fur ) instead. While Gory Fur is also not being by ( Mangle ).
